closed/src/java.base/share/classes/jdk/crypto/jniprovider/NativeCrypto.java

```diff
@@ -70,6 +77,88 @@ public static final int getVersion() {
         return ossl_ver;
     }
 
+    /**
+     * Check whether native crypto is enabled. Note that, by default, native
+     * crypto is enabled (the native crypto library implementation is used).
+     *
+     * The property 'jdk.nativeCrypto' is used to disable all native cryptos
+     * (Digest, CBC, GCM, RSA, ChaCha20, EC, and PBE), while the given
+     * property should be used to disable the given native crypto algorithm.
+     *
+     * @param property the property used to disable the given algorithm
```

Comment:
I'm not a fan of the word "disable" in these comments. While, I suppose there's nothing _technically_ wrong, I suggest the phrase "used to control enablement" is preferable to "used to disable".

Reply:
Thank you. Although it's unlikely to be used, someone could say `-Djdk.nativeCrypto=true`, which would *not* disable use of the native implementation.